src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star__blok">
+      <div class="star__line stars--0">
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--1">
+        <div class="star__style star--active"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--2">
+        <div class="star__style star--active"></div>
+        <div class="star__style star--active"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
+        <div class="star__style star--pasive"></div>
```

Comment:
The class `star--pasive` should be corrected to `star--passive` to fix the spelling error.